src/App.js

```diff
@@ -5,14 +5,47 @@ import Header from './components/header/header.component';
 import Homepage from './pages/homepage/homepage.component';
 import MapPage from './pages/map/map.component';
 import AboutPage from './pages/about/about.component';
-
 import './App.css';
 
+// npm install react-particles-js
+import Particles from 'react-particles-js';
+import 'tachyons';
+import '../node_modules/font-awesome/css/font-awesome.min.css';
```

Comment:
we don't import anything directly in node_modules, please read on https://www.digitalocean.com/community/tutorials/how-to-use-font-awesome-5-with-react to see how to use fontawesome with a react project.